vms/volunteer/utils.py

```diff
@@ -0,0 +1,19 @@
+from functools import wraps
+from django.shortcuts import render
+from django.http import Http404
+from volunteer.services import get_volunteer_by_id
+
+def vol_id_check(func):
+    @wraps(func)
+    def wrapped_view(request, volunteer_id):
+        vol = getattr(request.user, 'volunteer', hasattr(request.user, 'administrator'))
+        if not vol:
```

Comment:
Well.. in my local db, there is a user who is neither an admin nor a volunteer. ie, only a `django.contrib.auth.models.User`. It happens when you add a user by `python manage.py createsuperuser` or like accessing the `/admin` page and adding only a user.